src/index.html

```diff
@@ -1,13 +1,282 @@
+          <div class="categories__products ">
+            <a href="#" class="categories__products-1">
+              <img
+                src="./images/categories/birds.png"
+                alt="birds"
+                class="categories__image categories__image--img-after-tablet"
+              >
+              <img
+                src="./images/categories/birds-mobile.svg"
+                alt="birds"
+                class="categories__image categories__image--img-before-tablet"
+              >
+            </a>
+            <a href="#" class="categories__products-2">
+              <img
+                src="./images/categories/ear.svg"
+                alt="ear"
+                class="categories__image"
+              >
+            </a>
+          </div>
+          <h3 class="categories__description">All products</h3>
+        </article>
+        <article class="categories__section">
+          <div class="categories__products">
+            <a href="#" class="categories__products-3">
+              <img
+                src="./images/categories/one-ear-piece.svg"
+                alt="one ear piece"
+                class="categories__image"
+              >
+            </a>
+            <a href="#" class="categories__products-4">
+              <img
+                src="./images/categories/ear-stick.png"
+                alt="ear stick"
+                class="categories__image categories__image--img-after-tablet"
+              >
+              <img
+                src="./images/categories/ear-stick-mobile.svg"
+                alt="ear stick"
+                class="categories__image categories__image--img-before-tablet"
+              >
+            </a>
+          </div>
+          <h3 class="categories__description">Audio</h3>
+        </article>
+        <article class="categories__section">
+          <div class="categories__products">
+            <a href="#" class="categories__products-5">
+              <img
+                src="./images/categories/phone.png"
+                alt="phone"
+                class="categories__image categories__image--img-after-tablet"
+              >
+              <img
+                src="./images/categories/phone-mobile.svg"
+                alt="phone"
+                class="categories__image categories__image--img-before-tablet"
+              >
+            </a>
+            <a href="#" class="categories__products-6">
+              <img
+                src="./images/categories/charger.svg"
+                alt="charger"
+                class="categories__image"
+              >
+            </a>
+          </div>
+          <h3 class="categories__description">Accessories</h3>
+        </article>
+        <a href="#" class="categories__view-all">View all</a>
+      </section>
+
+      <section class="about-us" id="about-us">
+        <img
+          src="./images/about-us/bg-before-about-us.svg"
+          alt="background"
+          class="about-us__bg"
+        >
```

Comment:
The height of this element doesn't match the figma design

Yours:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/0659ec43-ab6e-40c7-a0de-9c4e68f43192)

Figma:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/a0409d2e-360a-4f4e-a399-e092c55ced95)

Bear in mind this height will change depending on the screen resolution breakpoint for improved responsiveness:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/49f21d9f-4f80-41e2-87e9-ab6e434f4b7b)